upstream/upstream_test.go

```diff
@@ -175,10 +175,10 @@ func (suite *ListSuite) TestHealthcheck() {
 		for i := 0; i < 10; i++ {
 			backend, err := l.Pick()
 			if err != nil {
-				return retry.UnexpectedError(err)
+				return err
```

Comment:
was this intentional?

Reply:
yep, it's actually something that linter catches - all errors are unexpected by default, so no need to wrap them once again